libraries/AP_BattMonitor/AP_BattMonitor_INA2xx.cpp

```diff
@@ -146,11 +145,33 @@ bool AP_BattMonitor_INA2XX::write_word(const uint8_t reg, const uint16_t data) c
 
 void AP_BattMonitor_INA2XX::timer(void)
 {
+    // allow for power-on after boot
+    if (!configured) {
+        uint32_t now = AP_HAL::millis();
+        if (now - last_configure_ms > 200) {
```

Comment:
This should probably be higher. The SMBus timeout is specified as 35 milliseconds, doing this at 5Hz would be 17.5% bus utilization, and could be pretty terrible for other sensors on the same bus.

Reply:
I take it back, this isn't SMBus so it's not a factor, sorry!